android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/register/PatientRegisterActivity.kt

```diff
@@ -69,8 +69,8 @@ class PatientRegisterActivity : BaseRegisterActivity() {
         getCustomNavigationOptions().navigationOptions.map {
           NavigationMenuOption(
             id = it.id.hashCode(),
-            title = it.title,
-            iconResource = this.getDrawable(it.icon)
+            title = it.title ?: "",
```

Comment:
can we move this logic into `NavigationMenuOption`? that is let it handle a null value here